### ISSUE TITLE
Is there a way to manipulate image color?

### ISSUE THREAD

**Issue by babyrusa** [iyegoroff/react-native-color-matrix-image-filters#38]
For example if my image is white (actual photo of an object), I want to change the object color to blue, or yellow, etc (give a HEX color or an RGB color).
I tried the RGBA object but it fills the entire picture with the color and you cannot see the object anymore.
Thank you

**Comment by iyegoroff** [comment]
ColorTone & DuoTone filters directly take HEX colors. You can find their implementations [here](https://github.com/iyegoroff/rn-color-matrices/blob/085b3082c89af0625aa8e88b447cf335ba0822bf/index.ts#L329-L356). In general custom color manipulations can be done with ColorMatrix filter.

**Comment by babyrusa** [comment]
@iyegoroff  do u mind explaining how ColorTone works. I notice ColorTone does part of what I wanted, but the color isn't complete correct. I guess it's because it takes lightColor and darkColor (which I don't understand exactly those are).
But I happened to come across ur another library https://github.com/iyegoroff/react-native-image-filter-kit and tried RBGA function there and I got what I wanted too.
